assets/src/edit-story/components/library/panes/text/karma/textPane.karma.js

```diff
@@ -40,6 +40,13 @@ describe('CUJ: Creator can Add and Write Text: Consecutive text presets', () =>
     fixture = new Fixture();
     fixture.setFlags({ enableSmartTextColor: true });
     await fixture.render();
+
+    await fixture.editor.library.textTab.click();
+    // Give some time for the things to be available for the tests.
+    await fixture.events.sleep(300);
+    await waitFor(() =>
+      expect(fixture.editor.canvas.framesLayer.frames[0].node).toBeTruthy()
```

Comment:
Hm, eslint doesn't like an `expect()` inside the `beforeEach`, which also does seem to violate some principles.

Could we just remove the expect and `waitFor(() => fixture.editor.canvas.framesLayer.frames[0].node)`?

Reply:
Oops, copying issues. Will change.